DATA/production/configurations/CTFSkimming/ctf-skim-workflow.sh

```diff
@@ -77,6 +77,13 @@ has_detector_ctf CTP && add_W o2-ctp-entropy-encoder-workflow "--select-ir-frame
 
 add_W o2-ctf-writer-workflow "--output-dir $CTF_DIR --min-file-size ${CTF_MINSIZE} --max-ctf-per-file ${CTF_MAX_PER_FILE} --onlyDet $WORKFLOW_DETECTORS_CTF $CTF_MAXDETEXT"
 
+if [[ $ALIEN_JDL_CPUCORES == 8 ]]; then
+  export MULTIPLICITY_PROCESS_tpc_entropy_decoder=2
+  MULTIPLICITY_PROCESS_tpc_entropy_encoder=2
+else
+  export TIMEFRAME_RATE_LIMIT=2
+fi
+
```

Comment:
No, the whole setting should go before the workflow composition, e.g. to https://github.com/AliceO2Group/O2DPG/blob/da75bf959363d9f944a144c72d7acdbe940460a7/DATA/production/configurations/CTFSkimming/ctf-skim-workflow.sh#L14 and the TIMEFRAME_RATE_LIMIT should be always set. I.e.
```
TIMEFRAME_RATE_LIMIT=2
if [[ $ALIEN_JDL_CPUCORES == 8 ]]; then
  export MULTIPLICITY_PROCESS_tpc_entropy_decoder=2
  export MULTIPLICITY_PROCESS_tpc_entropy_encoder=2
  export TIMEFRAME_RATE_LIMIT=3
fi
```
What I am not sure is who sets the NUMAID for the throttling. I remember on the grid it is set as a job ID to avoid conflicts.
Perhaps @catalinristea knows?

Reply:
Ah yes, sorry, the settings go above, I was too quick. 
Why do you need the NUMAID?

Reply:
Ciao,

I'm only aware of OPTIMIZED_PARALLEL_ASYNC=pp_4gpu, which I think does the switch to full NUMA domain. But probably this isn't what you need.

Yours, Catalin.

Reply:
I moved the lines, can you check now?

Reply:
Hi,
NUMAID is added as an argument of the ` --timeframes-rate-limit-ipcid`, apparently it is converted to grid job ID internally.
I've added some fixes.
Cheers,
 Ruben

Reply:
Hi,
No, since we add the option, we must provide also the value. Ok, let's use 0 as I've added it until it is proven that it is wrong.